docs/user-guide/sync-options.md

```diff
@@ -180,6 +180,10 @@ metadata:
     argocd.argoproj.io/sync-options: Force=true,Replace=true
 ```
 
+!!! warning
+      Setting `Force=true` is equivalent to using the `--force` flag in `kubectl`.
+      This may have unintended consequences for some resources!
```

Comment:
I would recommend to link or clarify those unintended consequences.

Reply:
Thanks, I agree! Have fixed this now.